test/bats/e2e-provider.bats

```diff
@@ -160,10 +160,15 @@ export VALIDATE_TOKENS_AUDIENCE=$(get_token_requests_audience)
 }
 
 @test "CSI inline volume test with pod portability - read secret from pod" {
+  # wait for secrets to mount
+  sleep 180
   wait_for_process $WAIT_TIME $SLEEP_TIME "kubectl exec secrets-store-inline-crd -- cat /mnt/secrets-store/$SECRET_NAME | grep '${SECRET_VALUE}'"
 
   result=$(kubectl exec secrets-store-inline-crd -- cat /mnt/secrets-store/$SECRET_NAME)
   [[ "${result//$'\r'}" == "${SECRET_VALUE}" ]]
+
+  sleep 10
+  archive_info
```

Comment:
was this for debugging?

Reply:
Yes, I think lets keep it as if something goes wrong or encounter  some flakiness, this will be helpful.